populate/src/main/resources/seed/portals/demo/studies/heartdemo/emails/adminNotificationTest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "stableId": "hd_hd_adminKitReceived",
+  "name": "Kit Received",
```

Comment:
I don't think this name/stableId matches the content

Reply:
oops